card/lib/card/config/environments/test.rb

```diff
@@ -1,5 +1,5 @@
 # -*- encoding : utf-8 -*-
-Decko.application.class.configure do
+Card::Application.configure do
```

Comment:
This variation loads, but even though it looks like seed -t is loading fixtures, no test data (load of joe user fails in early test setup). I think this pattern works.